docs/book/src/user/quick-start.md

```diff
@@ -73,14 +73,36 @@ curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-aws" asset:"in
 {{#/tab }}
 {{#tab vSphere}}
 
-Check the [vSphere provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases) for an up-to-date components file.
-
-For more information about prerequisites, credentials management, or permissions for vSphere, visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/docs/getting_started.md).
+It is required to use an official CAPV machine image for your vSphere VM templates. See [Uploading CAPV Machine Images](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/docs/getting_started.md#uploading-the-capv-machine-image) for instructions on how to do this.
 
 ```bash
-kubectl create -f {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-vsphere" asset:"infrastructure-components.yaml" version:"0.5.x"}}
+# Upload vCenter credentials as a Kubernetes secret
+$ cat <<EOF | kubectl apply -f -
```

Comment:
The infrastructure-components deployments would hang without the secret, didn't want to leave room for confusion because a user thinks applying infrastrucutre-components.yaml should just work.